### ISSUE TITLE
您好，这几个测试后的参数是什么意思？

### ISSUE THREAD

**Issue by ChenShuai-CA** [VDIGPKU/IterNet#5]
eval loss = 2.528, ccr = 0.595, cwr = 0.125, ted = 16603.000, ned = 5534, ted/w = 1.216
你好，我用Evaluation的命令测试测试集，为什么得到的这几个参数没有说明是什么？我不明白这几个参数的意义。为什么没有accuracy这个指标？

**Comment by chuxiaojie** [comment]
参照[评价指标](https://github.com/FangShancheng/ABINet/issues/66)
ccr是字符准确率，cwr单词准确率，ed是编辑距离，ned是正规化编辑距离
